views/default/forms/poll/edit.php

```diff
@@ -67,6 +67,25 @@
 ?>
 
 <div>
+<?php
+
+$options_values = array();
+for ($i = 1; $i < 11; $i++) {
```

Comment:
I agree with iionly that 11 is a bit arbitrary limit. I'd suggest adding a text field that allows poll owner to enter a number. The saving action can then check the given limit and show an error if the limit is equal or greater than the amount of entered options.